### ISSUE TITLE
Suggestion: descriptions for all types

### ISSUE THREAD

**Issue by dylang** [apollographql/eslint-plugin-graphql#62]
Suggestion:

`TypeDescriptions`: `Always`

Checks that `types` always have descriptions.

`bad`
```graphql 
type Widget {
  id: ID
  name: String
}
```

`good`
```graphql
# Blocks of content shown on the home page
type Widget {
  id: ID
  name: String
}
```

**Comment by stubailo** [comment]
Right now this tool doesn't really lint the schema files, only queries. I think a tool to lint the schema would be very welcome but would probably live in a separate repository?

If you're interested in working on that, let's get it going!
